traefik/templates/deployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
       serviceAccountName: {{ include "traefik.serviceAccountName" . }}
       terminationGracePeriodSeconds: 60
       hostNetwork: {{ .Values.hostNetwork }}
+      initContainers:
```

Comment:
Shoudln't line 55 and 56 be swapped? Like, the initContainer should only be there if "configured"?

Reply:
Good catch.